2022-03-02 11:20:58,153 INFO    MainThread:3029 [wandb_setup.py:_flush():75] Loading settings from /Users/wangyangwu/.config/wandb/settings
2022-03-02 11:20:58,153 INFO    MainThread:3029 [wandb_setup.py:_flush():75] Loading settings from /Users/wangyangwu/Documents/Maastro/NeuralNets/BasicUNet/wandb/settings
2022-03-02 11:20:58,153 WARNING MainThread:3029 [wandb_setup.py:_flush():75] Unknown environment variable: WANDB_CONFIG_DIR
2022-03-02 11:20:58,153 INFO    MainThread:3029 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-02 11:20:58,153 INFO    MainThread:3029 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-02 11:20:58,153 INFO    MainThread:3029 [wandb_init.py:_log_setup():386] Logging user logs to /Users/wangyangwu/Documents/Maastro/NeuralNets/BasicUNet/wandb/run-20220302_112058-380hp1q5/logs/debug.log
2022-03-02 11:20:58,153 INFO    MainThread:3029 [wandb_init.py:_log_setup():387] Logging internal logs to /Users/wangyangwu/Documents/Maastro/NeuralNets/BasicUNet/wandb/run-20220302_112058-380hp1q5/logs/debug-internal.log
2022-03-02 11:20:58,154 INFO    MainThread:3029 [wandb_init.py:init():420] calling init triggers
2022-03-02 11:20:58,154 INFO    MainThread:3029 [wandb_init.py:init():425] wandb.init called with sweep_config: {}
config: {}
2022-03-02 11:20:58,155 INFO    MainThread:3029 [wandb_init.py:init():441] re-initializing run, found existing run on stack: 192in8jc
2022-03-02 11:20:58,162 INFO    MainThread:3029 [wandb_run.py:finish():1468] finishing run wwy/Basic_UNet_['CTV1']/192in8jc
2022-03-02 11:20:58,176 INFO    MainThread:3029 [jupyter.py:save_history():429] not saving jupyter history
2022-03-02 11:20:58,176 INFO    MainThread:3029 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-02 11:20:58,176 INFO    MainThread:3029 [wandb_init.py:_jupyter_teardown():318] cleaning up jupyter logic
2022-03-02 11:20:58,176 INFO    MainThread:3029 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-03-02 11:20:58,178 INFO    MainThread:3029 [wandb_run.py:_restore():1769] restore
2022-03-02 11:20:59,181 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 640
  total_bytes: 640
}

2022-03-02 11:20:59,637 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 640
  total_bytes: 640
}

2022-03-02 11:21:00,638 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 640
  total_bytes: 9590
}

2022-03-02 11:21:00,746 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 640
  total_bytes: 9590
}

2022-03-02 11:21:00,852 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 640
  total_bytes: 9590
}

2022-03-02 11:21:00,958 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 640
  total_bytes: 9590
}

2022-03-02 11:21:01,064 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 640
  total_bytes: 9590
}

2022-03-02 11:21:01,169 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 640
  total_bytes: 9590
}

2022-03-02 11:21:01,276 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 640
  total_bytes: 9590
}

2022-03-02 11:21:01,385 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 640
  total_bytes: 9590
}

2022-03-02 11:21:01,490 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 640
  total_bytes: 9590
}

2022-03-02 11:21:01,596 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 640
  total_bytes: 9590
}

2022-03-02 11:21:01,702 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 640
  total_bytes: 9590
}

2022-03-02 11:21:01,807 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 640
  total_bytes: 9590
}

2022-03-02 11:21:01,913 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 640
  total_bytes: 9590
}

2022-03-02 11:21:02,019 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9590
  total_bytes: 9590
}

2022-03-02 11:21:02,130 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9590
  total_bytes: 9590
}

2022-03-02 11:21:02,239 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9590
  total_bytes: 9590
}

2022-03-02 11:21:02,348 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9590
  total_bytes: 9590
}

2022-03-02 11:21:02,455 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9590
  total_bytes: 9590
}

2022-03-02 11:21:02,563 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9590
  total_bytes: 9590
}

2022-03-02 11:21:02,668 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9590
  total_bytes: 9590
}

2022-03-02 11:21:02,774 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9590
  total_bytes: 9590
}

2022-03-02 11:21:03,616 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9590
  total_bytes: 9590
}

2022-03-02 11:21:04,503 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9590
  total_bytes: 9590
}
local_info {
}

2022-03-02 11:21:05,685 INFO    MainThread:3029 [wandb_run.py:_append_files():2194] logging synced files
2022-03-02 11:21:05,693 INFO    MainThread:3029 [wandb_init.py:init():471] starting backend
2022-03-02 11:21:05,693 INFO    MainThread:3029 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-03-02 11:21:05,695 INFO    MainThread:3029 [backend.py:ensure_launched():219] starting backend process...
2022-03-02 11:21:05,702 INFO    MainThread:3029 [backend.py:ensure_launched():224] started backend process with pid: 3104
2022-03-02 11:21:05,704 INFO    MainThread:3029 [wandb_init.py:init():480] backend started and connected
2022-03-02 11:21:05,717 INFO    MainThread:3029 [wandb_run.py:_label_probe_notebook():921] probe notebook
2022-03-02 11:21:05,717 INFO    MainThread:3029 [wandb_run.py:_label_probe_notebook():931] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-02 11:21:05,717 INFO    MainThread:3029 [wandb_init.py:init():550] updated telemetry
2022-03-02 11:21:05,720 INFO    MainThread:3029 [wandb_init.py:init():581] communicating current version
2022-03-02 11:21:06,587 INFO    MainThread:3029 [wandb_init.py:init():586] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-02 11:21:06,589 INFO    MainThread:3029 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-02 11:21:06,976 INFO    MainThread:3029 [wandb_init.py:init():624] starting run threads in backend
2022-03-02 11:21:10,540 INFO    MainThread:3029 [wandb_run.py:_console_start():1827] atexit reg
2022-03-02 11:21:10,541 INFO    MainThread:3029 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.WRAP
2022-03-02 11:21:10,541 INFO    MainThread:3029 [wandb_run.py:_redirect():1738] Wrapping output streams.
2022-03-02 11:21:10,541 INFO    MainThread:3029 [wandb_run.py:_redirect():1762] Redirects installed.
2022-03-02 11:21:10,541 INFO    MainThread:3029 [wandb_init.py:init():651] run started, returning control to user process
2022-03-02 11:21:10,541 INFO    MainThread:3029 [wandb_run.py:_config_callback():966] config_cb None None {'epoch': 1, 'batch_size': 1, 'dropout_rate': 0.5, 'learning_rate': 0.1}
2022-03-02 11:21:10,561 INFO    MainThread:3029 [wandb_watch.py:watch():43] Watching
